modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/model/handlers.scala

```diff
@@ -36,6 +36,7 @@ import scalaz._
 import Scalaz._
 
 object handlers {
+  private val VoidEffect = Effect(Future(NoAction: Action))
 
   trait Handlers {
     implicit def pfMonoid[A, B]: Monoid[PartialFunction[A, B]] = new Monoid[PartialFunction[A, B]] {
```

Comment:
It took me a minute to convince myself that this was a `Monoid` as I wasn't convinced that it was associative, but I think I convinced myself that it is.

Reply:
I plan to add tests for these once we move to cats. that would verify that monoid laws are upheld

Reply:
👍